auth/server/src/main/java/com/seagates3/controller/PolicyController.java

```diff
@@ -160,8 +162,15 @@ public ServerResponse create() {
      */
     @Override public ServerResponse list() {
       List<Policy> policyList = null;
+      Map<String, Object> parameters = new HashMap<>();
+      if (requestBody.containsKey("OnlyAttached")) {
```

Comment:
move onlyAttached as constant. If they are being used only inside this function then it should be constant inside this function. Else it should be class level constant

Reply:
fixed